packages/backend-common/src/reading/GithubUrlReader.test.ts

```diff
@@ -230,4 +243,91 @@ describe('GithubUrlReader', () => {
       );
     });
   });
+
+  describe('readTree', () => {
+    it('returns the files from a archive', async () => {
+      const worker = setupServer();
```

Comment:
Maybe mirror what the test-utils do and have this setup/teardown up in the describe block? https://github.com/backstage/backstage/blob/master/packages/test-utils/src/testUtils/msw/index.ts#L18